Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM hydroshare/hs_docker_base:release-1.9
+FROM hydroshare/hs_docker_base:django-1.11
```

Comment:
Once hs_docker_base review is approved, I'll merge, tag, build the new image and update version to the new release version here.

Reply:
hs_docker_base has been merged and tagged.  Once dockerhub is finished building the new image, I'll update this tag to release-1.9.1 and it'll be ready to review.